buildenv/jenkins/jobs/pipelines/Pipeline_Build_Template.groovy

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+*******************************************************************************/
+
+/**
+ * A template that defines a build job.
+ */
+
+if (!binding.hasVariable('SDK_VERSION')) SDK_VERSION = ""
+if (!binding.hasVariable('PLATFORM')) PLATFORM = ""
+
+def GIT_URI = 'https://github.com/samuel-rubin/openj9.git'
+def GIT_BRANCH = '*/jenkins_automatic_pipeline'
+
+pipelineJob("$JOB_NAME") {
+    description('<h1>THIS IS AN AUTOMATICALLY GENERATED JOB DO NOT MODIFY, IT WILL BE OVERWRITTEN.</h1><p>This job is defined in Pipeline_Build_Template.groovy in the openj9 repo, if you wish to change it modify that</p>')
+    definition {
+        cpsScm {
+            scm {
+                git {
+                    remote {
+                        url(GIT_URI)
+                    }
+                    branch("${GIT_BRANCH}")
+                    extensions {
+                        cleanBeforeCheckout()
+                    }
+                }
+            }
+            scriptPath('buildenv/jenkins/jobs/builds/Build-Test-Any-Platform')
+            lightweight(true)
+        }
+    }
+    logRotator {
+        numToKeep(30)
+    }
+    parameters {
+        choiceParam('SDK_VERSION', ["${SDK_VERSION}"])
+        choiceParam('PLATFORM', ["${PLATFORM}"])
+        stringParam('OPENJDK_REPO')
+        stringParam('OPENJDK_BRANCH')
+        stringParam('OPENJDK_SHA')
+        stringParam('OPENJ9_REPO')
+        stringParam('OPENJ9_BRANCH')
+        stringParam('OPENJ9_SHA')
+        stringParam('OMR_REPO')
+        stringParam('OMR_BRANCH')
+        stringParam('OMR_SHA')
+        stringParam('ADOPTOPENJDK_REPO')
+        stringParam('ADOPTOPENJDK_BRANCH')
+        stringParam('VARIABLE_FILE')
+        stringParam('VENDOR_REPO')
+        stringParam('VENDOR_BRANCH')
+        stringParam('VENDOR_CREDENTIALS_ID')
```

Comment:
These 4 should have default values which are configured global Jenkins variables.
`VARIABLE_FILE_DEFAULT` for example